api/actions/__test__/createWorkItem.spec.js

```diff
@@ -197,219 +245,224 @@ const reqWithGusTitlePrefix = {
 };
 
 describe('createGusItem action', () => {
```

Comment:
Updated all the tests to check for the GUS API interaction rather than the old Connect API push. Added couple of pin-down tests for the bug I fixed.